addons/web/static/src/views/relational_model.js

```diff
@@ -453,10 +453,14 @@ export class Record extends DataPoint {
     }
 
     get evalContext() {
-        return {
+        const evalContext = {
             // ...
             ...this.dataContext,
         };
+        if (this.resId) {
+            evalContext.active_id = this.resId;
```

Comment:
In BasicModel, I see that we add other keys in the eval context:
```js
{
                active_id: evalContext.id || false,
                active_ids: evalContext.id ? [evalContext.id] : [],
                active_model: element.model,
                current_company_id,
                id: evalContext.id || false,
}
```
What about these?